lib/fortune.js

```diff
@@ -316,8 +339,43 @@ Fortune.prototype.resource = function(name, schema, options, schemaCallback) {
     authMethods: authMethods,
     validation: validation
   };
+
   plugins.init(this, this._resources[name]);
+
   hooks.initGlobalHooks(_this._resources[name], _this.options);
+
+  var customHooks = _.flatten(_.map(_.keys(customTypes), function(key) {
+    // Modify a names of the hooks to include field name they applied to
+    _.each(customTypes[key].hooks, function(whenHooks, when) {
```

Comment:
Ok, think i got it. In design doc these hooks were passed through as properties of options, but chaining is probably even better.